### PR TITLE
fix: table selection, improve filtering in cmdK, issue in billing

### PR DESCRIPTION
**Pull request by umihai29**
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build related changes
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe below)

### CODE REVIEW

File: packages/apps/frontera/src/routes/src/components/CommandMenu/commands/organization/AddNewOrganizations.tsx [hunk 1]
```diff
@@ -75,15 +72,6 @@ export const AddNewOrganization = observer(() => {
     store.ui.commandMenu.toggle('AddNewOrganization');
   };
 
-  useKeyBindings(
```
Comment (Kasia9090) [inline packages/apps/frontera/src/routes/src/components/CommandMenu/commands/organization/AddNewOrganizations.tsx]: why this change? User should be able to submit even if neither of inputs are focused

File: packages/apps/frontera/src/routes/src/components/RootSidenav/RootSidenav.tsx [hunk 2]
```diff
@@ -51,7 +50,7 @@ export const RootSidenav = observer(() => {
         handleItemClick={handleItemClick}
         togglePreference={togglePreference}
       />
-      <UserActionSection />
+      {/* <UserActionSection /> */}
```
Comment (Kasia9090) [inline packages/apps/frontera/src/routes/src/components/RootSidenav/RootSidenav.tsx:53]: why it's commented out? will it be restored?

Comment (umihai29) [reply packages/apps/frontera/src/routes/src/components/RootSidenav/RootSidenav.tsx:53]: steyn wasn't sure we gonna added back or not so i left commented

File: packages/apps/frontera/src/routes/src/components/CommandMenu/commands/organization/AddNewOrganizations.tsx (hunk 1 above)
Comment (umihai29) [reply packages/apps/frontera/src/routes/src/components/CommandMenu/commands/organization/AddNewOrganizations.tsx]: by mistake i added back

**Review by Kasia9090 — APPROVED** [review]
